src/storage/mvcc/txn.rs

```diff
@@ -276,13 +295,23 @@ impl<S: Snapshot, P: PdClient + 'static> MvccTxn<S, P> {
                 "async commit is not yet compatible with large transactions"
```

Comment:
It's not zero in pessimistic transactions because the min_commit_ts is `max(prewrite's min_commit_ts, pessimistic lock's min_commit_ts)`.

Reply:
ahh.. I forgot pessimistic locks also have min_commit_ts.